packages/app/src/app/pages/Sandbox/Editor/Workspace/screens/ProjectInfo/index.tsx

```diff
@@ -0,0 +1,174 @@
+    event.preventDefault();
+
+    if (customTemplate) {
+      deleteTemplate();
+    } else {
+      modalOpened({ modal: 'deleteSandbox' });
+    }
+  };
+
+  return (
+    <>
+      <Collapsible title="Sandbox Info" defaultOpen>
+        <Stack direction="vertical" gap={6}>
+          <Element style={{ padding: `0 ${theme.space[3]}` }}>
+            <Title editable />
+            <Description editable />
+          </Element>
+          <Stack
+            gap={2}
+            align="center"
+            marginBottom={4}
+            style={{ padding: `0 ${theme.space[3]}` }}
+          >
+            <Avatar user={author} /> <Text>{author.username || 'Anon'}</Text>
+          </Stack>
+          <Stats sandbox={currentSandbox} />
+          <List>
+            <ListItem justify="space-between">
+              <Text as="label" htmlFor="frozen">
+                Frozen
+              </Text>
+              <Switch
+                id="frozen"
+                onClick={updateFrozenState}
+                on={customTemplate ? sessionFrozen : isFrozen}
+              />
+            </ListItem>
+            <ListItem justify="space-between">
+              <Text> {forkedTemplateSandbox ? 'Template' : 'Forked From'}</Text>
+              <Link>
+                {getSandboxName(forkedFromSandbox || forkedTemplateSandbox)}
+              </Link>
+            </ListItem>
+            <ListItem justify="space-between">
+              <Text>Environment</Text>
+              <Link>{template}</Link>
+            </ListItem>
+          </List>
+        </Stack>
+      </Collapsible>
+      <Collapsible title="Privacy" defaultOpen>
+        <Stack
+          direction="vertical"
+          gap={4}
+          style={{ padding: `0 ${theme.space[3]}` }}
+        >
+          <Select
+            disabled={!isPatron}
+            icon={GlobeIcon}
+            onChange={({ target: { value } }) =>
+              sandboxPrivacyChanged({
+                privacy: Number(value) as 0 | 1 | 2,
+                source: 'sidebar',
+              })
+            }
+            value={privacy}
+          >
+            <option value={0}>Public</option>
+
+            {isPatron && (
+              <option value={1}>Unlisted (only available by url)</option>
+            )}
+
+            {isPatron && <option value={2}>Private</option>}
+          </Select>
+          {!isPatron ? (
+            <Text variant="muted" size={2}>
+              You an change privacy of a sandbox as a Pro.
+              <br />
+              Become a Pro.
+            </Text>
+          ) : null}
+        </Stack>
+      </Collapsible>
+
+      <Element marginX={2} marginY={4}>
+        <Button variant="secondary">Save as template</Button>
+      </Element>
+
+      <Stack
+        justify="center"
+        style={{ position: 'absolute', width: '100%', bottom: theme.space[3] }}
+      >
+        <DeleteButton
+          // @ts-ignore
+          onClick={onDelete}
+          variant="link"
```

Comment:
This is a button which is difficult to understand, because some of the styles come from `variant` and the rest comes from styled().

After we figure out what's up with the css prop, I'd write this as

<Button variant="link" css={{ ':hover, :focus': .... }}>

Reply:
Yeah, that's what you had, it's still like this in the components package so as soon as the css thing is fixed we revert it and delete the custom component